src/overlays/actors/ovl_Lamp_Light/ef_lamp_light.c

```diff
@@ -19,21 +25,114 @@ ActorProfile Lamp_Light_Profile = {
     /* */ Ef_Lamp_Light_actor_ct,
     /* */ Ef_Lamp_Light_actor_dt,
     /* */ Ef_Lamp_Light_actor_move,
-    /* */ (ActorFunc)none_proc1,
+    /* */ (void*)none_proc1,
     /* */ NULL,
 };
-#endif
+static s16 D_80967884_jp[4] = { 2, 2, 1, 0 };
+static s16 D_8096788C_jp[4] = { 0x10, 0x10, 8, 0 };
+static s16* D_80967894_jp[2] = { D_80967884_jp, D_8096788C_jp };
+static s16 D_8096789C_jp[4] = { 0, 0, 0, 0 };
+static s16 D_809678A4_jp[4] = { 0xC8, 0xC8, 0x96, 0 };
+static s16* D_809678AC_jp[2] = { D_8096789C_jp, D_809678A4_jp };
+
+void Ef_Lamp_Light_actor_ct(Actor* thisx, Game_Play* game_play) {
+    s32 pad UNUSED;
+    Lamp_Light* this = (Lamp_Light*)thisx;
+
+    game_play->kankyo.unk_1C = &this->unk_178;
+    Light_diffuse_ct(&this->unk_178, 0, 0x50, 0, (u8)0, (u8)0, (u8)0);
+    this->unk_188 = Global_light_list_new(game_play, &game_play->glight, &this->unk_178);
+}
+
+void Ef_Lamp_Light_actor_dt(Actor* thisx, Game_Play* game_play) {
+    Lamp_Light* this = (Lamp_Light*)thisx;
+
+    game_play->kankyo.unk_1C = NULL;
+    Global_light_list_delete(&game_play->glight, this->unk_188);
+}
+
+s32 func_80967644_jp(void) {
```

Comment:
```suggestion
s32 eLL_get_light_sw_other_room(void) {
```

Reply:
should i change the other ones from `Ef_Lamp_Light` to `eLL` as well? like `Ef_Lamp_Light_actor_ct` to `eLL_actor_ct`

Reply:
They're genuinely the original names lol, no don't change it